services/authService.ts

```diff
@@ -0,0 +1,13 @@
+import { Login } from "../model/auth";
+
+const axios = require('axios');
+
+module.exports.login =async function (login: Login): Promise<void> {
```

Comment:
aren't you supposed to return the token?

Reply:
All fixed, thank you!